app/addons/replication/controller.js

```diff
@@ -186,25 +208,79 @@ export default class ReplicationController extends React.Component {
           max={600}
           startValue={300}
           stepSize={60}
-          onPoll={Actions.getReplicationActivity}
+          onPoll={this.getAllActivity.bind(this)}
           />
         <RefreshBtn
-          refresh={Actions.getReplicationActivity}
+          refresh={this.getAllActivity.bind(this)}
           />
       </div>
     );
   }
 
+  getTabElements () {
+    const {tabSection} = this.state;
+    const elements = [
+      <TabElement
+        key={1}
+        selected={tabSection === 'activity'}
+        text={"Replicator DB Activity"}
+        onChange={this.onTabChange.bind(this, 'activity', '#/replication')}
+      />
+    ];
+
+    if (this.state.supportNewApi) {
+      elements.push(
+        <TabElement
+          key={2}
+          selected={tabSection === '_replicate'}
+          text={"_replicate Activity"}
+          onChange={this.onTabChange.bind(this, '_replicate', '#/replication/_replicate')}
+        />
+      );
+    }
+
+    return elements;
+  }
+
+  onTabChange (section, url) {
+    Actions.changeTabSection(section, url);
+  }
+
+  getCrumbs () {
+    if (this.state.tabSection === 'new replication') {
+      return [{'name': 'Job Configuration'}];
+    }
+
+    return [];
+  }
+
+  getTabs () {
+    if (this.state.tabSection === 'new replication') {
+      return null;
+    }
+
+    return (
+      <TabElementWrapper>
+        {this.getTabElements()}
+      </TabElementWrapper>
+    );
+  }
+
   render () {
+    const { checkingAPI } = this.state;
+
+    if (checkingAPI) {
```

Comment:
Is it worth switching this to `this.state.checkingAPI` to eliminate the previous line?  Doesn't look like we use `checkingAPI` anywhere else later on.